Bottom: 7cdb1e45ea500e1abcac8bd3b591fc3dd536c1db
Top:    51ef6ad1c26da2b22baa5d114c02de4cd91f104c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-16 15:21:50 -0500

Refresh of all-docker_volume

---
diff --git a/docker/docker-mediawiki/mediawiki.sh b/docker/docker-mediawiki/mediawiki.sh
index d52bd31..be0f4f0 100755
--- a/docker/docker-mediawiki/mediawiki.sh
+++ b/docker/docker-mediawiki/mediawiki.sh
@@ -53,8 +53,8 @@ done
 # make certian the containers exist
 docker inspect ${WIKI_CONTAINER_NAME} > /dev/null
 docker inspect ${WIKI_DB_CONTAINER_NAME} > /dev/null
-docker inspect ${WIKI_DV_NAME} > /dev/null
-docker inspect ${WIKI_DB_DV_NAME} > /dev/null
+docker volume inspect ${WIKI_DV_NAME} > /dev/null
+docker volume inspect ${WIKI_DB_DV_NAME} > /dev/null
 
 get_db_user_and_password() {
     db_user="$(docker 2>&1 exec ${WIKI_CONTAINER_NAME} grep -e '^$wgDBuser' LocalSettings.php|sed 's|^.* = \"\(.*\)\";$|\1|' || true)"
